packages/calling-component-bindings/src/captionsSelector.ts

```diff
@@ -122,6 +122,16 @@ export const _captionsBannerSelector: _CaptionsBannerSelector = reselect.createS
     // Following Teams app logic, no matter how many 'Partial' captions come,
     // we only pick first one according to start time, and all the other partial captions will be filtered out
     // This will give customers a stable captions experience when others talking over the dominant speaker
+    // First turn all partial captions that are older than 10 seconds to final
```

Comment:
Can there be multiple unfinalized captions? Just wondering what would break if instead of waiting 10/15 seconds, we just say if there's a newer unfinalized caption then mark older ones as final?